packages/blockchain-wallet-v4-frontend/src/data/auth/sagas.ts

```diff
@@ -718,13 +718,20 @@ export default ({ api, coreSagas, networks }) => {
   const triggerWalletMagicLink = function* (action) {
     const formValues = yield select(selectors.form.getFormValues('login'))
     const { step } = formValues
+    const legacyMagicEmailLink = (yield select(
+      selectors.core.walletOptions.getFeatureLegacyMagicEmailLink
+    )).getOrElse(false)
```

Comment:
i think we want to default to true right? `getOrElse(true)`. at least for the timebeing